tests/test.sh

```diff
@@ -95,7 +95,10 @@ test_pbf() {
 
   if [[ $OSTYPE == linux* ]]; then
     ./tests/fixtures/vtzero-check "$FILENAME"
-    ./tests/fixtures/vtzero-show "$FILENAME" > "$FILENAME.txt"
+    # see https://gdal.org/en/stable/programs/ogrmerge.html#ogrmerge
+    ogrmerge.py -o "$FILENAME.geojson" "$FILENAME" -single -src_layer_field_name "source_mvt_layer" -src_layer_field_content "{LAYER_NAME}" -f "GeoJSON" -overwrite_ds
```

Comment:
for users which don't have the right dependency installed this would look like the tests not being able to run.
For the other dependencys, we can reasonably assume that they are already insstalled ^^

As a fresh dev, a missing python file might look like a bug and lead to me skipping testing..

=> Could you add a check at the start of the test that bails if `ogrmerge.py` is not installed and advises the dev to do so?